Makefile

```diff
@@ -185,8 +185,10 @@ $(PHP_CS_FIXER): Makefile
 $(PHPSTAN): vendor
 
 $(INFECTION): vendor $(shell find bin/ src/ -type f) $(BOX) box.json.dist .git/HEAD
+	composer require infection/codeception-adapter infection/phpspec-adapter
 	$(BOX) validate
 	$(BOX) compile
+	composer remove infection/codeception-adapter infection/phpspec-adapter
```

Comment:
without removing adapters, Travis fails with weird errors during `bin/infection ...`

I wasn't able understand why

* locally works as expected
* tried debugging by ssh'ing into container im Travis - random fails

Reply:
It says InitialTestsFailed. Can we assume that this is a mistake? E.g. tests are not broken.

Reply:
ok, I admit that the machine is smarter than me. It fails locally as well with unit tests for known reasons. Tests assume that classes from adapters do not exist.

Anyway, removing adapters is ok for me here.